### PR TITLE
PHPUnit forward compatibility layer

### PR DESCRIPTION
**Pull request by MarioBlazek**

### CODE REVIEW

**Comment by MarioBlazek** [comment]
@pspanja WDYT ?

File: composer.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
         "ezsystems/ezpublish-kernel": "^6.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4.7"
+        "phpunit/phpunit": "^4.8.35|^5.7"
```
Comment (pspanja) [inline composer.json:17]: Do we need `4.8.35` for some reason, shouldn't `^4.8|^5.7` be enough?

Comment (MarioBlazek) [reply composer.json:17]: in 4.8.35 `TestCase` class was introduced.

Comment (MarioBlazek) [reply composer.json:17]: https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-4.8.md#4835---2017-02-06

Comment (pspanja) [reply composer.json:17]: OK then :)

**Comment by pspanja** [comment]
Looks good, thanks @MarioBlazek!

**Pull request merged**